cmd/flow-collector/main.go

```diff
@@ -283,6 +355,33 @@ func main() {
 		w.WriteHeader(http.StatusNotFound)
 	}))
 
+	var userApi = api1.PathPrefix("/user").Subrouter()
+	userApi.StrictSlash(true)
+	userApi.HandleFunc("/", authenticated(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
```

Comment:
Differently from the other functions, here I don't reach 'handler.go' as I don't need to use the Flow.Collector @ajssmith